docs/resources/Guild.md

```diff
@@ -61,25 +61,25 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 
 ###### Default Message Notification Level
 
-| Key           | Value |
-| ------------- | ----- |
-| ALL_MESSAGES  | 0     |
-| ONLY_MENTIONS | 1     |
+| Key           | Value | Description                                                                        |
+| ------------- | ----- | ---------------------------------------------------------------------------------- |
+| ALL_MESSAGES  | 0     | members will receive notifications for all messages by default                     |
+| ONLY_MENTIONS | 1     | members will receive notifications only for messages that @mention them by default |
 
 ###### Explicit Content Filter Level
 
-| Level                 | Integer |
-| --------------------- | ------- |
-| DISABLED              | 0       |
-| MEMBERS_WITHOUT_ROLES | 1       |
-| ALL_MEMBERS           | 2       |
+| Level                 | Integer | Description                                                     |
+| --------------------- | ------- | --------------------------------------------------------------- |
+| DISABLED              | 0       | no media content will be scanned                                |
+| MEMBERS_WITHOUT_ROLES | 1       | media content sent by members without any roles will be scanned |
+| ALL_MEMBERS           | 2       | media content sent by any member will be scanned                |
 
 ###### MFA Level
 
-| Level    | Integer |
-| -------- | ------- |
-| NONE     | 0       |
-| ELEVATED | 1       |
+| Level    | Integer | Description                                        |
+| -------- | ------- | -------------------------------------------------- |
+| NONE     | 0       | guild has no MFA/2FA requirement                   |
```

Comment:
This could be a bit more specific. `guild does not have a MFA/2FA for moderator actions`